ansible/tasks/docker/setup.yml

```diff
@@ -21,7 +21,6 @@
       - systemtap-sdt-dev
       - ccache
       - clang
-      - flex
```

Comment:
@michelp have moved this over [here](https://github.com/supabase/postgres/blob/49563a6eb20a82446117d171637741018813e9ca/ansible/Dockerfile#L693). Also, is this a Docker only dependency or should this also be included in [this file](https://github.com/supabase/postgres/blob/49563a6eb20a82446117d171637741018813e9ca/ansible/tasks/postgres-extensions/29-pg_tle.yml)?